docs/ADR-EXPRESS-VS-ANGULAR.md

```diff
@@ -0,0 +1,70 @@
+
+**App 1: Angular**
+
+- Repository: [https://github.com/pcraig3/angular-ssr-demo](https://github.com/pcraig3/angular-ssr-demo)
+- Link: [http://angular-ssr-demo.canadacentral.azurecontainer.io:4000/](http://angular-ssr-demo.canadacentral.azurecontainer.io:4000/)
+
+**App 2: Express**
+
+- Repository: [https://github.com/pcraig3/expressbase](https://github.com/pcraig3/expressbase)
+- Link: [http://expressbase.canadacentral.azurecontainer.io:3005/](http://expressbase.canadacentral.azurecontainer.io:3005/start)
+
+While both apps were able to fulfill the above requirements, the difference was clear.
+
+### Angular: the wrong tool for the job
+
+Angular is a client-side JavaScript framework that is optimized for fairly complex apps with real-time data requirements and/or dynamic interactions. For our form-based web application, we don’t benefit from the more dynamic features that Angular offers. Our pages are lightweight and don’t require real-time updates. In fact, real-time updates would likely lead to usability issues.
+
+Even where we could use dynamic on-page changes like form validation, we have found [during usability testing in previous projects that users can be discouraged by real-time error messages or miss them altogether](https://github.com/cds-snc/ircc-rescheduler/issues/92), which is supported by [GDS’ guidance on validation messages](https://design-system.service.gov.uk/components/error-message/#when-not-to-use-this-component).
+
+Additionally, building a purely client-side application introduces many [potential issues](https://blog.pope.tech/2018/10/25/angular-accessibility-and-you/) that aren’t present in server-rendered web applications. An example would be when the focus is lost between page transitions. While possible to solve, this is work isn’t needed in a server-rendered application. In general, it is easier to build an accessible form-based service with a server-rendered application.
+
+Unfortunately, building a server-rendered Angular app (using [Angular Universal](https://angular.io/guide/universal)) is not how this framework is intended to be used. In order to server-render the application, the complexity of the configuration increases dramatically, resulting in a more complicated application with a terrible development experience.
+
+Indeed, while CRA is looking to develop and support Angular applications in the future, CRA is **not** looking to develop and support _server-side rendered_ Angular applications. Server-side rendering (SSR) for better accessibility is a priorities for CDS, but not necessarily for CRA. Even if we went through the steep learning curve of an SSR Angular application, it is unlikely that CRA would agree to support it.
+
+### Express and Pug: well better
```

Comment:
**Express and Pug: well, better** 
_or_
**Express and Pug: well...better**

Again, this is just my preference for commas (or ellipses). So very much take it or leave it because it doesn't really matter 😂